db_script.sql

```diff
@@ -0,0 +1,9 @@
+INSERT INTO mysql.user (User,Host,authentication_string,ssl_cipher,x509_issuer,x509_subject) VALUES('demouser','localhost',PASSWORD('demopassword'),'','','');
```

Comment:
There's `CREATE USER` for simply user creation, but Top chose to use INSERT into user table.  This is not wrong but it looks like he thinks too complex for simply requirement but too simply for detailed requirement.  IMO 🤔